content/sensu-go/5.13/reference/assets.md

```diff
@@ -96,6 +96,186 @@ sensu-example-handler_1.0.0_linux_amd64
 └── include
 {{< /highlight >}}
 
+### Asset hello world example
+
+In this example, we'll run a script that outputs `Hello World`:
+
+{{< highlight bash >}}
+hello-world.sh
+
+#!/bin/sh
+
+STRING="Hello World"
+
+echo $STRING
+
+if [ $? -eq 0 ]; then
+    exit 0
+else
+    exit 2
+fi
+{{< /highlight >}}
+
+The first step is to ensure that our directory structure is in place. As noted in [Example structure](#example-structure), our script could live in three potential directories in the project: `/bin`, `/lib`, or `/include`. For this case, we will put our script in the `/bin` directory. We'll create the directories `sensu-go-hello-world` and `/bin`:
+
+{{< highlight bash >}}
+$ mkdir sensu-go-hello-world
+
+$ cd sensu-go-hello-world
+
+$ mkdir bin
+
+$ tree
+.
+└── bin
+    └── hello-world.sh
```

Comment:
For continuity, I suggest adding a command before `tree` showing that we copied the .sh into place